dev-libs/libusb/libusb-1.0.24.recipe

```diff
@@ -22,14 +22,14 @@ COPYRIGHT="2001 Johannes Erdfelt
 LICENSE="GNU LGPL v2.1"
 REVISION="1"
 SOURCE_URI="https://github.com/libusb/libusb/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="02620708c4eea7e736240a623b0b156650c39bfa93a14bcfa5f3e05270313eba"
+CHECKSUM_SHA256="b7724c272dfc5713dce88ff717efd60f021ca5b7c8e30f08ebb2c42d2eea08ae"
 SOURCE_FILENAME="libusb-$portVersion.tar.gz"
-PATCHES="libusb-$portVersion.patchset"
+#PATCHES="libusb-$portVersion.patchset"
```

Comment:
Unneded

Reply:
I verified that all prior Haiku-related libusb patches were integrated in this release.

Reply:
Having a quick look on their github page seems to confirm, no need for it then:
```diff
- #PATCHES="libusb-$portVersion.patchset"
```